suites/pacific/upgrades/tier-2_scaleup_scaledown_after_upgrade_4x-to-5x-container.yaml

```diff
@@ -0,0 +1,598 @@
+- test:
+    name: install ceph pre-requisites
+    module: install_prereq.py
+    config:
+      is_production: true
+    abort-on-fail: True
+
+- test:
+    name: ceph ansible install containerized rhcs 4.x from cdn
+    polarion-id: CEPH-83573588
+    module: test_ansible.py
+    config:
+      use_cdn: True
+      build: '4.x'
+      ansi_config:
+        ceph_origin: repository
+        ceph_repository: rhcs
+        ceph_repository_type: cdn
+        ceph_rhcs_version: 4
+        ceph_stable_release: nautilus
+        osd_scenario: lvm
+        osd_auto_discovery: False
+        ceph_stable_rh_storage: True
+        containerized_deployment: true
+        ceph_docker_image: "rhceph/rhceph-4-rhel8"
+        ceph_docker_image_tag: "latest"
+        ceph_docker_registry: "registry.redhat.io"
+        copy_admin_key: true
+        dashboard_enabled: True
+        dashboard_admin_user: admin
+        dashboard_admin_password: p@ssw0rd
+        grafana_admin_user: admin
+        grafana_admin_password: p@ssw0rd
+        node_exporter_container_image: registry.redhat.io/openshift4/ose-prometheus-node-exporter:v4.6
+        grafana_container_image: registry.redhat.io/rhceph/rhceph-4-dashboard-rhel8:4
+        prometheus_container_image: registry.redhat.io/openshift4/ose-prometheus:v4.6
+        alertmanager_container_image: registry.redhat.io/openshift4/ose-prometheus-alertmanager:v4.6
+        cephfs_pools:
+          - name: "cephfs_data"
+            pgs: "16"
+          - name: "cephfs_metadata"
+            pgs: "16"
+    desc: deploy ceph containerized 4.x cdn setup using ceph-ansible
+    destroy-cluster: False
+    abort-on-fail: true
+
+- test:
+    name: rados_bench_test
+    module: radosbench.py
+    config:
+      pg_num: '128'
+      pool_type: 'normal'
+    desc: run rados bench for 360 - normal profile
+
+- test:
+    name: check-ceph-health
+    module: exec.py
+    config:
+      cmd: ceph -s
+      sudo: True
+    desc: Check for ceph health debug info
+
+- test:
+    name: Parallel run
+    module: test_parallel.py
+    parallel:
+     - test:
+         name: rados_bench_test
+         module: radosbench.py
+         config:
+           pg_num: '128'
+           pool_type: 'normal'
+         desc: run rados bench for 360 - normal profile
+     - test:
+         name: rbd-io
+         module: rbd_faster_exports.py
+         config:
+           io-total: 100M
+           cleanup: false
+         desc: Perform export during read/write,resizing,flattening,lock operations
+     - test:
+         name: Upgrade containerized ceph to 5.x latest
+         polarion-id: CEPH-83573680
+         module: test_ansible_upgrade.py
+         config:
+           build: '5.x'
+           ansi_config:
+             ceph_origin: distro
+             ceph_stable_release: pacific
+             ceph_repository: rhcs
+             ceph_rhcs_version: 5
+             osd_scenario: lvm
+             osd_auto_discovery: False
+             ceph_stable: True
```

Comment:
Can we avoid upstream related settings in this section?

Reply:
Done.